src/Formula.elm

```diff
@@ -10,12 +10,18 @@ import Parser exposing
   )
 import Parser.LanguageKit exposing (variable, sequence, whitespace)
 
+type Term
+  = Var String
```

Comment:
There is no substantial difference between a constant and a free variables in tableaux. To keep things simple, I used only variables.